common/config/azure-pipelines/jobs/version-bump.yaml

```diff
@@ -137,8 +137,10 @@ jobs:
 
   - job: Bump
     displayName: Bump Version
-    dependsOn: CreateBranch
-    condition: in(dependencies.CreateBranch.result, 'Succeeded', 'Skipped')
+    dependsOn:
+    - CreateBranch
+    - checkPrevCommit
```

Comment:
The Previous Commit job is named "CheckPrevCommit", don't know if dependsOn is case sensitive, but should be consistent regardless. 

https://github.com/iTwin/itwinjs-core/blob/020769b2973cdd7b9d6bfb622ef4b9d34af03e1e/common/config/azure-pipelines/jobs/version-bump.yaml#L43